qa/tasks/vstart_runner.py

```diff
@@ -809,6 +795,16 @@ def run_ceph_w(self, watch_channel=None, shell=False):
                                    shell=shell)
         return proc
 
+    def do_rados(self, cmd, pool=None, namespace=None, remote=None, **kwargs):
+        args = ['rados', '--cluster', self.cluster]
+        if pool is not None:
+            args += ['--pool', pool]
+        if namespace is not None:
+            args += ['--namespace', namespace]
+        args.extend(cmd)
+        proc = self.controller.run(args=args, wait=False, **kwargs)
+        return proc
+
```

Comment:
You can do something like this and avoid duplicating `do_rados()` - https://github.com/ceph/ceph/pull/38481/files#diff-203644a2b9fd4a8cf54b0c52970ffbd1aebb87a134dcf9fc0dc88286da7d4ab6R1511.

I can do that too when I rebase my PR.

Reply:
Maybe do that in your PR is better. Will leave this to you.

Reply:
Okay.